debbuild

```diff
@@ -1428,25 +1585,26 @@ sub binpackage {
 
     # Munge things so that Debian tools don't choke on errant blank lines
     $pkgdata{$pkg}{description} =~ s/\s+$//g;	# Trim trailing blanks
-    $pkgdata{$pkg}{description} =~ s/^ $/ ./mg;	# Replace lines consisting of " \n" with " .\n"
+    $pkgdata{$pkg}{description} =~ s/^\s+$/ ./mg;	# Replace lines consisting of " \n" with " .\n"
 
     # Give an estimate of the installation size
     my ($installedsize) =
       qx($specglobals{__du} -s --apparent-size $specglobals{buildroot}/$pkg) =~
         /(\d+)/;
 
+    my $maintainer = defined $pkgdata{main}{packager} ? $pkgdata{main}{packager} : expandmacros('%{_deb_maintainer}');
```

Comment:
Please remove this. I don't want `%_deb_maintainer`. It is intentional that this is mapped to the `%packager` macro as they are equivalent.

Reply:
Hmm, need to check it.

Reply:
There is no `%packager` macro, it's actually `Packager` fielf of `spec` and it's not defined in 99% of the spec files.
`dpkg` is allowing to building `deb` with no Maintainer specified, but there are a lot of noise messages that Maintainer is missing in the package metadata on installing such packages. The only way is to specify `Packager` in `spec`, but in the project I made this changes for, we are trying to prevent any changes in `spec` as much as possible. It was the reason I implemented additional macro to use value from in case if `Packager` is not specified in a `spec`.

Reply:
The `%packager` macro is supposed to work to define the field if you don't set it in the spec. If that's not working, that's a separate bug.

Reply:
Debian effectively requires that information (hence the warnings when it's not there), so we throw warnings/errors on it missing. It seems that when the `Packager:` field isn't set or the `%packager` macro is not set, it'll get set to `Maintainer: %{packager}` due to failed expansion.